CONTRIBUTING/README.md

```diff
@@ -96,6 +87,8 @@ If you're making any updates or fixes that apply to the latest version of k6, yo
 
 This is to make sure that any changes you make are also brought over to the next major release version of k6.
 
+> If your update or fix also applies to previous versions of k6, use the `apply-patch` script to port your changes to the latest version, and two previous versions.
```

Comment:
I don't know if it's clear enough that this is something important for the reader... 🤔